dialplan/asterisk/extensions_lib_user.conf

```diff
@@ -20,6 +20,7 @@ same  =   n,Set(XIVO_DSTNUM=${IF(${EXISTS(${XIVO_DSTNUM})}?${XIVO_DSTNUM}:${XIVO
 same  =   n,Set(XIVO_CONTEXT=${IF(${EXISTS(${XIVO_CONTEXT})}?${XIVO_CONTEXT}:${XIVO_BASE_CONTEXT})})
 same  =   n,Set(WAZO_CHANNEL_DIRECTION=to-wazo)
 same  =   n,Set(__XIVO_CALLORIGIN=${IF(${EXISTS(${XIVO_CALLORIGIN})}?${XIVO_CALLORIGIN}:intern)})
+same  =   n,Set(__WAZO_CALL_DIRECTION=${IF(${EXISTS(${WAZO_CALL_DIRECTION})}?${WAZO_CALL_DIRECTION}:internal)})
```

Comment:
Suggestion: keep the variable absent if internal. When reading the variable, if it's absent, then the call is internal. That way, we don't have to add it to every dialplan (voicemail, conference, queue, etc.)